### PR TITLE
Improved Intro Articulation

### PR DESCRIPTION
**Pull request by zora89**
This repo deserves the best documentation standard. Hence, improved articulation to be more inviting and in flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,18 @@ https://user-images.githubusercontent.com/34342551/121651071-d9fd1580-cab7-11eb-
 
 </details></h2>
 
-## Why `ugit`
+## Why use `ugit`
```
Comment (Bhupesh-V) [inline README.md:40]:
```suggestion
## Why use `ugit`?
```

File: README.md [hunk 2]
```diff
@@ -37,18 +37,18 @@ https://user-images.githubusercontent.com/34342551/121651071-d9fd1580-cab7-11eb-
...
 
-- You ran an accidental `git` command you didn't want to.
-- You don't want to waste your time searching on _how to undo ..._
-- Because you want to focus on problems at hand and not on Git (You don't like context switching)
-- Because `ugit` is cool
+- You ran an accidental `git` command you wish to 'undo'.
+- You want to save time by not searching for _how to undo ..._
+- Your focus is on problems at hand and not on Git (avoid context switching)
+- Because `ugit` is precise & uber cool !
```
Comment (Bhupesh-V) [inline README.md:45]: ❤️

**Comment by Bhupesh-V** [comment]
@all-contributors please add @zora89  for doc

**Comment by zora89** [comment]
Thank you, I will dive deeper and certainly hope to add more value.


-
*Zorawar Purohit*
Virtual Meet: calendly.com/zpmeet
More Info: Bio <https://zorawar.bio.link/>

**Pull request merged**